test/controllers/friendships_controller_test.rb

```diff
@@ -0,0 +1,34 @@
+require 'test_helper'
```

Comment:
We will deal with friendship_controller_tests once we settle what needs to be done in the friendship_controller